Sources/Plasma/Apps/plClient/Mac-Cocoa/PLSLoginWindowController.mm

```diff
@@ -214,7 +214,7 @@ - (void)windowDidLoad
 
     [self.window center];
     [self.productTextField
-        setStringValue:[NSString stringWithSTString:plProduct::ProductString().c_str()]];
+        setStringValue:ST2NSString(plProduct::ProductString().c_str())];
```

Comment:
Another possible leak. stringWithSTString returned an autoreleased string. ST2NSString does not.